CONTRIBUTING.md

```diff
@@ -0,0 +1,79 @@
+# Contributing
+
+## Getting Started
+
+General information about setting up your Python environment, testing modules,
+Ansible coding styles, and more can be found in the [Ansible Community Guide](
+https://docs.ansible.com/ansible/latest/community/index.html).
+
+Information about boto library usage, module utils, testing, and more can be
+found in the [AWS Guidelines](https://docs.ansible.com/ansible/devel/dev_guide/platforms/aws_guidelines.html)
+documentation.
+
+## AWS Collections
+
+There are two related collections containing AWS content (modules and plugins).
+
+### amazon.aws
+This collection contains the `module_utils` (shared libraries) used by both collections.
+Content in this collection is included downstream in Red Hat Ansible Automation Platform.
+
+Code standards, test coverage, and other supportability criteria may be higher in this collection. 
+
+### community.aws
+This collection contains modules and plugins contributed and maintained by the Ansible AWS
+community.  The `community.aws` collection is tested and generally assured to work in
+conjunction with `amazon.aws`.
+
+New modules and plugins developed by the community should be proposed to `community.aws`.
+Content in this collection that is stable and meets other acceptance criteria has the potential
+to be promoted and migrated into `amazon.aws`.
+
```

Comment:
Can you evaluate if this link is appropriate here - https://docs.ansible.com/ansible/devel/community/contributing_maintained_collections.html#contributing-maintained-collections 

In theory, it's supposed to cover this collection (and I know you sent me some edits I need to implement on that page...if I can find them again!)  If you think it's appropriate, please add the link. If not, let's discuss what needs to change before you can add a link to it.

Reply:
Added it to the amazon.aws section, does that seem like the right place to you?  I'm duplicating this guide in the community collection in https://github.com/ansible-collections/community.aws/pull/109  so trying to think how it should be read overall.
(I'll resend you the edits, no worries!  :))